README.md

```diff
@@ -100,11 +98,46 @@ package { 'PSExcel-psc':
 }
 ```
 
+### The provider
+
+The provider to use will either be `windowspowershell` or `powershellcore`. Nodes using `powershell.exe` will use `windowspowershell`, and nodes that have PowerShell core (`pwsh.exe`) will use the `powershellcore` provider with both the `psrepository` and `package` types.
+
 ## Limitations
 
 Note that PowerShell modules can be installed side by side so installing a newer
 version of a module will not remove any previous versions.
 
+## Reference
+
+### Types
+
+### psrepository
+
+Allows you to specify and configure a repository. The type expects a valid OneGet package provider source over an HTTP or HTTPS url.
```

Comment:
The cmdlet will support it, but for the it will fail in the module because of the validation in [psrepository.rb](https://github.com/hbuckle/puppet-powershellmodule/blob/7b5f09687f4cc97bcf343d64aaa791791399c7e7/lib/puppet/type/psrepository.rb#L27)

Reply:
We're tracking this issue in [MODULES-6995](https://tickets.puppetlabs.com/browse/MODULES-6995)